modules/social_features/social_core/src/Entity/EntityAutocompleteMatcher.php

```diff
@@ -19,7 +19,7 @@ class EntityAutocompleteMatcher extends EntityAutocompleteMatcherBase {
   public function getMatches($target_type, $selection_handler, $selection_settings, $string = '') {
     $matches = [];
 
-    $options = [
+    $options = $selection_settings + [
       'target_type' => $target_type,
       'handler' => $selection_handler,
       'handler_settings' => $selection_settings,
```

Comment:
Wouldn't this be redundant in this case?

So it would be:

```
    $options = $selection_settings + [
      'target_type' => $target_type,
      'handler' => $selection_handler,
    ];
    $handler = $this->selectionManager->getInstance($options);
    ```

Reply:
Hi Ronald, thank you for your review, I just pushed a fix.